src/ontology/phenio.Makefile

```diff
@@ -66,8 +66,9 @@ missing_phenio_nodes.tsv: | $(TMPDIR)
 	curl -L https://data.monarchinitiative.org/monarch-kg-dev/latest/qc/missing_phenio_nodes.tsv --create-dirs --retry 4 --max-time 400 > $@
 
 # Just get the CHEBI IDs
+# and change them to URIs
 $(TMPDIR)/monarch_chebi_terms.txt: missing_phenio_nodes.tsv
-	cat $^ | grep CHEBI | cut -f 1 > $@
+	cat $^ | grep CHEBI | cut -f 1 | sed 's/CHEBI:/http:\/\/purl.obolibrary.org\/obo\/CHEBI_/' > $@
```

Comment:
no need for this, ROBOT can deal with CURIEs just fine

Reply:
Great, I thought it should be able to handle CURIEs.